### DISCUSSION TITLE
Using Enterprise Edition in the Docker Image

### DISCUSSION THREAD

**Discussion by gaedemischwarz** [oracle/docker-images#2193] [Q&A] [answered]
Hello,

i have trouble to start a Oracle 19.3 Enterprise in the Docker image. 

the Docker image Build is done with: ./buildContainerImage.sh -e -v 19.3.0
the startup if the Container is done by the following.

docker run -d --name ORACLEDB \
    -p 1521:1521 -p 5500:5500 \
    -e ORACLE_SID=ORACLEDB \
    -e ORACLE_PWD=@PwdOra14 \
    -e ORACLE_EDITION=enterprise \
    -e ORACLE_CHARSET=AL32UTF8 \
    -e ENABLE_ARCHIVELOG=TRUE \
    -v ora_data:/opt/oracle/oradata \
    oracle/database:19.3.0-ee

The Binaries should also have the Oracle Enterprise in it. but after the Startup i got the following.

"The datafiles being used were created with standard edition software home. Please pass -e ORACLE_EDITION=standard to the docker run cmd"

**Comment by yunus-qureshi** [comment]
@gaedemischwarz your volume ora_data has datafiles that were created with standard edition. Pass the same edition or specify a new SID to create a new enterprise edition DB

**Comment by gaedemischwarz** [comment, marked as answer]
ah my fault. thanks, i was using docker volumes.  And where not seeing the Files.
